src/DynamoCoreWpf/Views/PackageManager/InstalledPackagesView.xaml

```diff
@@ -89,12 +89,30 @@
                                                    FontSize="11"
                                                    Foreground="#CCC"></TextBlock>
                                         <TextBlock Text="{x:Static p:Resources.InstalledPackageViewPendingInstallButton}"
-                                                   Visibility="{Binding Path=Model.MarkedForUninstall, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}"
+                                                   Visibility="{Binding Path=Model.EnableOldMarkedForUnistallState, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}"
```

Comment:
Little confused here, why couldn't you simply change the implementation of the `MarkedForUninstall` property instead of introducing a new one?

Reply:
I introduced another property because `MarkedForUninstall` is used in other parts of Dynamo, not only used in the UI...and I did not want to break that existing functionality by introducing the DebugMode flag into the equation.
Basically, if I had done : `bool MarkedForUninstall { get {return DebugModes.IsActive() && isMarkedForUninstall; } }` then, while having the DebugMode active, the other parts of the Dynamo that use `MarkedForUninstall` would not work anymore.

Reply:
MarkedForUninstall...should probably be replaced by the PackageStates ...when the implementation is ready